client/js/widgets/label.js

```diff
@@ -26,7 +27,8 @@ export class Label extends Widget {
 
   applyDeltaToDOM(delta) {
     super.applyDeltaToDOM(delta);
-    if(delta.text !== undefined || delta.twoRowBottomAlign !== undefined) {
+    if(delta.text !== undefined || delta.twoRowBottomAlign !== undefined || delta.spellCheck !== undefined) {
+      this.input.setAttribute('spellcheck', this.get('spellCheck') === true)
```

Comment:
In fact, technically this should go into a separate check similar to delta.editable, because spellcheck changing doesn't require changing all of the input attributes.

I.e. after this if block

```js
if (delta.spellCheck !== undefined)
  this.input.setAttribute('spellCheck', this.get('spellCheck') === true);
```

Reply:
So before I do another commit, is this what you are suggesting"

```
 super.applyDeltaToDOM(delta);
    if(delta.text !== undefined || delta.twoRowBottomAlign !== undefined) {
      if(delta.spellCheck !== undefined) {
        this.input.setAttribute('spellcheck', this.get('spellCheck') === true)
      }
      this.input.value = this.get('text');
 ```

Reply:
No, like this:
```js
    super.applyDeltaToDOM(delta);
    if(delta.text !== undefined || delta.twoRowBottomAlign !== undefined) {
      this.input.value = this.get('text');
      ...
    }
    if(delta.spellCheck !== undefined) {
      this.input.setAttribute('spellcheck', this.get('spellCheck') === true)
    }
```

Reply:
The way this works is that any change in property value will be in the delta dictionary, so this is saying if the spellcheck value changed, then change the spellcheck attribute on the input element.

Reply:
I see it now.  Thanks for the explanation and help.